miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -814,13 +816,101 @@ HotTarget.sample = (function() {
               }),
           // Aliquot: Single Cell columns
           HotUtils.makeColumnForDecimal('Input into Library',
-              (show['Aliquot'] && config.targetSampleClass.alias.indexOf('Single Cell') != -1), 'inputIntoLibrary', 14, 10, false, false)];
+              (show['Aliquot'] && config.targetSampleClass.alias.indexOf('Single Cell') != -1), 'inputIntoLibrary', 14, 10, false, false),
+
+          // Distribution to collaborator or outside destination
+          {
+            header: 'Distributed',
+            data: 'distributed',
+            type: 'dropdown',
+            trimDropdown: false,
+            source: ['True', 'False'],
```

Comment:
Are these the best terms for these? Maybe _Sent_ and _On-site_. I dunno if Illinca has preferences.